spec/unit/hanami/cli/commands/gem/new_spec.rb

```diff
@@ -1350,4 +1350,28 @@ class App < Hanami::App
       )
     end
   end
+
+  it "initializes a git repository" do
+    expect(bundler).to receive(:install!)
+      .and_return(true)
+
+    expect(bundler).to receive(:exec)
+      .with("hanami install")
+      .and_return(successful_system_call_result)
+
+    expect(bundler).to receive(:exec)
+      .with("check")
+      .at_least(1)
+      .and_return(successful_system_call_result)
+
+    expect(system_call).to receive(:call).with("npm", ["install"])
```

Comment:
The fact that we have to have all these non-relevant setup lines just so we can test for `git init` feels like a bit of a small to me, and suggests we should probably improve the test setup for the `new` command at least. Doing that in this PR would make it much larger than it needs to be, so I'm OK for this to test to be as it is for the sake of keeping this PR to a smaller, focused change :)